### PR TITLE
PR: Add docs tutorials blogpost and change picture for juanis2112

### PR DESCRIPTION
**Pull request by juanis2112**

### CODE REVIEW

File: content/blog/tutorial-docs/contents.lr [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+pub_date: 2020-07-08 10:00
+---
+summary: I joined the Spyder team almost two years ago, and I never thought I was going to end up working on docs. Six months ago I started a project with CAM Gerlach and Carlos Cordoba to improve Spyder’s documentation. At first, I didn’t actually understand how important docs are for software, especially for open source projects. However, during all this time I’ve learned how documentation has a huge impact on the open-source community and I’ve been thankful to have been able to do this. But, from the beginning, I asked myself “why am I the ‘right person’ for this?”
+---
+body:
+
+*This blogpost was originally published on the [Quansight Labs website](https://labs.quansight.org/blog/2020/07/writing-docs-is-not-just-writing-docs/)*.
+
+I joined the Spyder team almost two years ago, and I never thought I was going to end up working on docs. 
+
```
Comment (juanis2112) [inline content/blog/tutorial-docs/contents.lr]:
```suggestion
```

File: content/blog/tutorial-docs/contents.lr [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+However, during all this time I’ve learned how documentation has a huge impact on the open-source community and I’ve been thankful to have been able to do this. 
+But, from the beginning, I asked myself “why am I the ‘right person’ for this?”
+
+Improving Spyder’s documentation started as part of a NumFOCUS Small Development Grant awarded at the end of last year. 
+The goal of the project was not only to [update the documentation](http://docs.spyder-ide.org/develop/) for Spyder 4, but also to make it more user-friendly, so users can understand Spyder’s key concepts and get started with it more easily.
+ 
+One of the main ideas for this project was to create a series of short video tutorials, explaining the basic functionality of Spyder and its most important panes, allowing users to learn how to use Spyder faster and easier.
+ 
+Carlos Cordoba, our lead maintainer, thought I was the perfect person for this project because of my “good communication and organization skills”, my “clear and fluent English” (his words) and my previous experience at video editing and recording, which I actually gained by recording singing videos during my “YouTuber” phase. 
+
```
Comment (juanis2112) [inline content/blog/tutorial-docs/contents.lr]:
```suggestion
```

File: content/blog/tutorial-docs/contents.lr [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+However, if I’m a completely new user and I don’t even know how to open it, where do I start? 	
+ 
+I planned these videos as a series of progressive steps that can get users from zero experience to actually working with Spyder. 
+Hence, I divided the videos into three sections, “First Steps with Spyder”, “Working with Spyder”, and “Building Projects with Spyder”. 
+Each section builds on the one before in a way that they are clear enough so that users can find their way through Spyder without knowing anything about it.
+ 
+The [“First Steps with Spyder”](https://www.youtube.com/playlist?list=PLPonohdiDqg9epClEcXoAPUiK0pN5eRoc) section, live now on our YouTube channel, has three videos that provide a starting point for new users before they even open Spyder for the first time. 
+The [first tutorial](https://youtu.be/E2Dap5SfXkI) covers different ways of opening Spyder, the basics of using its interface and an introduction to its four main panes, along with a quick look at the others so that users can get familiar with how Spyder is organized. 
+
+
```
Comment (juanis2112) [inline content/blog/tutorial-docs/contents.lr]:
```suggestion
```

File: content/blog/tutorial-docs/contents.lr [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+ 
+In the [second video](https://youtu.be/WV9bm4ey7Cg), users can learn the basics of using Spyder’s four main panes. 
+The goal is that after this video, users are able to open and edit a file in the Editor, run a script and find the output in Spyder’s IPython Console and execute basic Python commands. 
+They should be able to interact with the Variable Explorer to browse and edit the objects in the console, and use the Help pane to get documentation in two different ways.
+ 
+The [third video](https://youtu.be/-dARZBUDk_s) is meant to show users how to customize Spyder’s interface to start working with it in the way they feel more comfortable according to their preferences. 
+It teaches users how to change the font and the theme of Spyder and rearrange its panes to display only the ones that they want such that it is easier for them to work. 
+
+
```
Comment (juanis2112) [inline content/blog/tutorial-docs/contents.lr]:
```suggestion
```

File: content/blog/tutorial-docs/contents.lr [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+It teaches users how to change the font and the theme of Spyder and rearrange its panes to display only the ones that they want such that it is easier for them to work. 
+
+ 
+After these three videos, I learned that one of the most important things for writing documentation, more than having a lot of experience with the software, is empathy. 
+Usually being a developer makes it hard to put yourself in the position of a user and understand exactly what a user needs. 
+Now, I realize that this is what actually makes me the “perfect person” for the project. 
+As a junior developer, without as much experience with Spyder, I was able to think more like new users and develop content in a way that they feel closer to us developers. 
+In the end, as Melissa said in her [blog post](https://labs.quansight.org/blog/2020/03/documentation-as-a-way-to-build-community/), I learned writing documentation is also a way of building community. 
+
+
```
Comment (juanis2112) [inline content/blog/tutorial-docs/contents.lr]:
```suggestion
```

**Review by CAM-Gerlach — APPROVED** [review]
LGTM, thanks @juanis2112

**Pull request merged**